capactio/website
# Prepare documentation for Dashboard

## Description

Changes proposed in this pull request:

- Prepare Overview document for Dashboard
- Include Dashboard in E2E architecture document

**Additional scope:**
- Prepare Action management document

- Refactor redirects script to make it easier to maintain redirects
   - Update release script 
   - See how it looked before refactor: https://github.com/capactio/website/pull/94/commits/23f1a3d761b8f6d3e19fa6a581513f3cb6990356

- [x] create follow-up task for refactoring existing docs to show Dashboard as an alternative: #94 

## Testing

See new docs and test redirections on https://ui-docs.capact-website.pages.dev/docs/next/dashboard-ui/overview

To test new redirections after release:
1. Checkout this branch
2. Edit `hack/make-release.sh` script and comment line 24 (`git push origin main`)
3. Run:
```
RELEASE_VERSION=0.6.0 ./hack/make-release.sh
```
4. See the last commit
5. Run `npm run build` and `npm run serve` and test redirects

## Related issue(s)

Resolves capactio/capact/issues/599
